docroot/modules/custom/nvli_custom_search/src/Plugin/Block/CustomSearchBlock.php

```diff
@@ -32,7 +32,11 @@ protected function blockAccess(AccountInterface $account) {
    * {@inheritdoc}
    */
   public function build($keyword = NULL) {
+    $build = [];
     $keyword = \Drupal::request()->get('keyword');
-    return \Drupal::formBuilder()->getForm('Drupal\nvli_custom_search\Form\CustomSearchForm', $keyword);
+    $build['#cache']['max-age'] = 0;
```

Comment:
@piyuesh23 
No in this form we are rendering the keyword in textfield. So if this is cached then keyword is not updating .
So we need this.